### ISSUE TITLE
Custom primary key confusion

### ISSUE THREAD

**Issue by simon-weber** [chrisglass/django_polymorphic#66]
With a set of models like:
```
Owner(polymorphic), has many Projects
FooOwner(Owner), has custom primary key

Project(polymorphic), has many Owners
BarProject(Project), has custom primary key
```

performing `barproject.owners.add(FooOwner())` seemed to have no effect; afterwards, `barproject.owners.all()` would be empty.

When I went to write a test case to recreate, I noticed I was triggering [this line](https://github.com/chrisglass/django_polymorphic/blob/b73f22f4e0a98f1e0c65625b0293c65b533bfaef/polymorphic/tests.py#L377) in my test output. I then removed the custom primary keys from FooOwner and BarProject, which fixed the problem.

Is this a known issue? The closest mention I saw in the docs was about [unique primary keys in calls to extra()](https://django-polymorphic.readthedocs.org/en/latest/advanced.html#about-queryset-methods), but the test output seems to suggest somebody is aware of this.

**Comment by vdboor** [comment]
This issue could be fixed now that #119 is merged, can you verify? (will release a new version on PyPI asap)

**Comment by simon-weber** [comment]
Cool; thanks. It'll take me a bit to dig up this old code, so if you're eager to release you might not want to wait on me :grin:

**Comment by vdboor** [comment]
hehe :-) a 0.7 release has just been published to support Django 1.7/1.8. I'm looking forward to the fixes you can provide.
